userpanel/modules/finances/style/bclean/templates/userbalancebox.html

```diff
@@ -1,7 +1,18 @@
 {box title="Your balance"}
 {$disable_invoices = ConfigHelper::checkConfig('userpanel.disable_invoices')}
-<style>
 
+<script>
+	$(function() {
+		$(".table-click").click(function() {
+			window.document.location = $(this).attr("data-href");
+		});
+	});
+</script>
+
+<style>
+	.table-click tr :hover, td :hover {
```

Comment:
Skąd tu się wzięła nazwa klasy `.table-click` dla elementu zewnętrznego w stosunku do `tr` skoro to dla `tr` nadajesz klasę `table-click`?

Reply:
Jak być powinno poprawnie? Nie wgłębiałem się jak to działa zastosowany przykład z internetu zadziałał więc nie modyfikowałem dalej.

Reply:
A czyli metodą szympansa robiłeś (ja to tak nazywam - czasem jak jestem mocno zmęczony też tej metody używam). Dajesz szympansowi kostki z literkami i czekasz aż ułoży z nich wyraz - jest to prawdopodobnie, choć bardzo małe szanse na to :rofl:

Poczytaj o selektorach fajny artykulik: https://www.w3schools.com/cssref/css_selectors.asp
Szkoda, żebym się pocił. Jak to zgłębisz to potem i w css/less poradzisz i w selekcji elementów przez jQuery.

Reply:
Zastosowane rozwiązanie działa w pełni ale jest niekoszerne.Ja ten artykuł zapisuje sobie na border i wolnej chwili go odczytam. Jesteś w stanie to poprawić tak jak uważasz i puścić PR?

Reply:
Zobacz czy po moim zmianach sprzed chwili też Ci działa.
No i użycie `black` jest nieeleganckie...

Reply:
No i teraz nie ma sposobu na to, żeby zaznaczyć dany wiersz checkboksem celem wydruku zbioru faktur - od razu faktura się otwiera...

Reply:
Moim zdaniem guzik "Drukuj" jest jednak bardziej intuicyjny. W tej chwili klient nie będzie wiedział jak wydrukować fakturę  i jak przez przypadek kliknie to będzie zdziwiony, że coś się dziwnego dzieje...

Reply:
Może lepiej pominąć aktywność dla ostatnich dwóch komórek wiersza i do tego wydruk faktury robić w nowym oknie, a nie w tym samym? Sam nie wiem - jakoś topornie to działa.

Reply:
1. Lepiej wydruk w nowym oknie.
2. Przycisk drukuj może być ale tylko ten zbiorczy na dole i kolumna z checboxami, żeby wydrukować tylko i wyłącznie wybrane FV.
3. Przycisk drukuj przy każdej FV ... IMO zbędna sprawa jeśli wiersz podświetla się tak ładnie.

Reply:
> Może lepiej pominąć aktywność dla ostatnich dwóch komórek wiersza i do tego wydruk faktury robić w nowym oknie, a nie w tym samym? Sam nie wiem - jakoś topornie to działa.

Dla ostatniej komórki można by pominąć.

Reply:
Wydaje mi się, że dla guzika "Drukuj" również. I najlepiej niech on zostanie, żeby można było konwencjonalnie drukować.

Reply:
Może ten przycisk mógłby się pokazywać tylko w momencie najechania na wiersz?

Reply:
To nie ładne rozwiązanie, które doprowadzi do `pływania` interfejsu użytkownika.